app/cdap/components/GridTable/components/Breadcrumb/styles.ts

```diff
@@ -25,6 +25,12 @@ export const useStyles = makeStyles({
     alignItems: 'center',
     marginRight: '30px',
     marginLeft: '34px',
+    '& .MuiBreadcrumbs-li': {
+      fontSize: '14px',
```

Comment:
no need to write numeric values as strings, just convert fontSize: '14px', to fontSize: 14,

Reply:
done